app/main.py

```diff
@@ -1,3 +1,61 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: "Distance") -> "Distance":
```

Comment:
Add more descriptive annotation to `other`. It can also be of type `int` or `float`.

Why are you using quotes for "Distance" everywhere? You can do it without them example: `Dinstance`
(Import annotations from future for this)

Fix in all cases